Sources/MapboxNavigation/ExitView.swift

```diff
@@ -145,6 +145,7 @@ public class ExitView: StylableView {
      This generates the cache key needed to hold the `ExitView`'s `imageRepresentation` in the `ImageCache` caching engine.
      */
     static func criticalHash(side: ExitSide,
+                             styleID: String?,
```

Comment:
Add `styleID` into the cache key, to avoid the delay of `style` update of `ExitView` and `GenericRouteShield`.

Reply:
Is it enough information to reliably store `ExitView` in cache without collisions?

Reply:
I think so, right now the delay of style change in `ExitView` and `GenericRouteShield` is caused by the retrieval of the key to find the right one. The  `styleID` would match them with exact style and solve this issue. Even if they cannot find it, they could create a new correct one.